packages/reports/app/styles/navi-reports/components/common-actions/schedule-modal.scss

```diff
@@ -0,0 +1,57 @@
+/**
+* Copyright 2021, Yahoo Holdings Inc.
+* Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+*/
+.modal.is-active {
+  visibility: visible !important;
+}
```

Comment:
```suggestion
```

Reply:
Without this on the reports page we still have visibility issues. How was it resolved in directory?